aws-android-sdk-core/src/main/java/com/amazonaws/auth/CognitoCredentialsProvider.java

```diff
@@ -451,8 +451,8 @@ public Date getSessionCredentialsExpiration() {
 
     @Deprecated
     /** Use {@link #getSessionCredentialsExpiration()} instead. */
-    public Date getSessionCredentitalsExpiration() {
-        return getSessionCredentialsExpiration();
+    public void getSessionCredentitalsExpiration() {
```

Comment:
Hmm, why change the return type here?  I think the way you had it before was correct.

Reply:
Oops! sorry I misunderstood Jameson's PR comment.